IOSExercises/Views/ColorsTableViewController.m

```diff
@@ -0,0 +1,64 @@
+//
+//  ColorsTableViewController.m
+//  IOSExercises
+//
+//  Created by Arnold Joseph Caesar Esteban on 11/27/20.
+//  Copyright © 2020 Arnold Joseph Caesar Esteban. All rights reserved.
+//
+
+#import "ColorsTableViewController.h"
+#import "EXRColor.h"
+
+static NSString *kCellIdentifier = @"colorCell";
+
+@implementation ColorsTableViewController {
+    NSArray<EXRColor *> *_availableColorOptions;
+}
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [self.tableView registerClass:UITableViewCell.class
+           forCellReuseIdentifier:kCellIdentifier];
+}
+
+#pragma mark - Table view delegate
+
+- (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath {
+    NSString *chosenColor = [_availableColorOptions objectAtIndex:indexPath.item].colorName;
+    NSLog(@"ColorsTableViewController: User chose color %@",  chosenColor);
+    [self.delegate sendSelectedColorName:chosenColor];
```

Comment:
If "send" means to send the color from one class to another, then possibly can find a more apt term to name the method with. For delegate methods especially, can name according to action, eg `didSelectColor:`